python/mxnet/ndarray/numpy/linalg.py

```diff
@@ -352,3 +352,58 @@ def slogdet(a):
     (1., -1151.2925464970228)
     """
     return _npi.slogdet(a)
+
+
+def solve(a, b):
+    r"""
+    Solve a linear matrix equation, or system of linear scalar equations.
+
+    Computes the "exact" solution, x, of the well-determined, i.e., full rank, linear matrix equation ax = b.
+
+    Parameters :
+    a : (..., M, M) ndarray
```

Comment:
Indent. And same for all the following doc. Please follow the same format as in

https://github.com/numpy/numpy/blob/0725a58415e68eab9fa7d7389cdb40716447a4fb/numpy/linalg/linalg.py#L326-L381